x-pack/plugins/observability_solution/observability/public/components/alerts_table/common/render_cell_value.tsx

```diff
@@ -71,63 +69,66 @@ const getRenderValue = (mappedNonEcsValue: any) => {
+  data,
   setFlyoutAlert,
   observabilityRuleTypeRegistry,
 }: {
-  setFlyoutAlert: (alertId: string) => void;
-  observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry;
-}): ReturnType<GetRenderCellValue> => {
-  return ({ columnId, data }) => {
-    if (!data) return null;
-    const mappedNonEcsValue = getMappedNonEcsValue({
-      data,
-      fieldName: columnId,
-    });
-    const value = getRenderValue(mappedNonEcsValue);
+  columnId: string;
+  data?: Array<{ field: string; value: any }>;
+  setFlyoutAlert?: (alertId: string) => void;
+  observabilityRuleTypeRegistry?: ObservabilityRuleTypeRegistry;
+}) => {
+  if (!data) return null;
+  const mappedNonEcsValue = getMappedNonEcsValue({
+    data,
+    fieldName: columnId,
+  });
+  const value = getRenderValue(mappedNonEcsValue);
 
-    switch (columnId) {
-      case ALERT_STATUS:
-        if (value !== ALERT_STATUS_ACTIVE && value !== ALERT_STATUS_RECOVERED) {
-          // NOTE: This should only be needed to narrow down the type.
-          // Status should be either "active" or "recovered".
-          return null;
-        }
-        return <AlertStatusIndicator alertStatus={value} />;
-      case TIMESTAMP:
-        return <TimestampTooltip time={new Date(value ?? '').getTime()} timeUnit="milliseconds" />;
-      case ALERT_DURATION:
-        return asDuration(Number(value));
-      case ALERT_SEVERITY:
-        return <AlertSeverityBadge severityLevel={value ?? undefined} />;
-      case ALERT_EVALUATION_VALUE:
-        const valuesField = getMappedNonEcsValue({
-          data,
-          fieldName: ALERT_EVALUATION_VALUES,
-        });
-        const values = getRenderValue(valuesField);
-        return valuesField ? values : value;
-      case ALERT_REASON:
-        const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
-        const alert = parseAlert(observabilityRuleTypeRegistry)(dataFieldEs);
+  switch (columnId) {
+    case ALERT_STATUS:
+      if (value !== ALERT_STATUS_ACTIVE && value !== ALERT_STATUS_RECOVERED) {
+        // NOTE: This should only be needed to narrow down the type.
+        // Status should be either "active" or "recovered".
+        return null;
+      }
+      return <AlertStatusIndicator alertStatus={value} />;
+    case TIMESTAMP:
+      return <TimestampTooltip time={new Date(value ?? '').getTime()} timeUnit="milliseconds" />;
+    case ALERT_DURATION:
+      return asDuration(Number(value));
+    case ALERT_SEVERITY:
+      return <AlertSeverityBadge severityLevel={value ?? undefined} />;
+    case ALERT_EVALUATION_VALUE:
+      const valuesField = getMappedNonEcsValue({
+        data,
+        fieldName: ALERT_EVALUATION_VALUES,
+      });
+      const values = getRenderValue(valuesField);
+      return valuesField ? values : value;
+    case ALERT_REASON:
+      const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
+      if (!observabilityRuleTypeRegistry) return <>{value}</>;
```

Comment:
Looks like git picked up this whole `switch` block because it changed indentation, but I *think* this is the only line that is actually different between the old switch and this one (this line has been added)